firmware/ventilator-controller-stm32/Core/Test/Pufferfish/Util/vector.txt

```diff
@@ -0,0 +1,60 @@
+Scenario: The function in Util Vectors works correctly
```

Comment:
Tests in this file are on the methods in the `Util::Vector` class, so the wording of the scenarios should match; you can also define scenarios by functionality, but it doesn't make sense to say "the function in Util push_back". Valid ways to write scenarios include: `Scenario: simple methods of the Vector class work correctly`, or `Scenario: insertion and deletion of elements in Vector at arbitrary indices works correctly`, or `Scenario: copying of elements to/from Vectors works correctly`